modules/kubernetes/pages/k8s-operator/installation.adoc

```diff
@@ -0,0 +1,43 @@
+= Installing TigerGraph Kubernetes Operator
+:description: Instructions on how to install TigerGraph Kubernetes Operator.
+:sectnums:
+
+This page describes the steps to install TigerGraph Kubernetes Operator on your Kubernetes cluster.
+
+The Operator only needs to be installed once for each cluster.
+If you run into the CRD
+
+== Prerequisites
+* You have configured the `kubectl` command-line tool for access to your running Kubernetes cluster.
+* Helm is installed on your local machine.
+* https://cert-manager.io/docs/installation/kubectl/[cert-manager is installed on your Kubernetes cluster].
+* If you are pulling from a private registry, configure your cluster for access to the registry.
+You can do this by https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret[creating a Secret] and https://kubernetes.io/docs/concepts/configuration/secret/#use-case-as-container-environment-variables[use it as a container environment variable].
+
+== Procedure
+
+[#_install_the_tigergraph_kubectl_plugin]
+=== Install the TigerGraph `kubectl` plugin
+Run the command below to install TigerGraph plugin for the `kubectl` commandline tool:
+
+[.wrap,console]
+----
+
+----
+
+
+=== Initialize TigerGraph Kubernetes Operator
+Run the command below to initialize the TigerGraph Kubernetes Operator.
+
+[.wrap,console]
+----
+$ kubectl tg init --helm-repo http://docker.tigergraph.com
+----
+
+If you are using a private helm repository, replace `http://docker.tigergraph.com` with the URL of your repository, and supply the name of your Secret:
```

Comment:
should we stress the secret of docker image registry?

Reply:
If someone is using a private helm repository, I think it's safe to assume that they have knowledge of how to access helm. I think I will add a note about how Secret is namespace-specific since that was something we struggled with.